apps/nextjs/src/components/assistant/assistant-panel.module.css

```
@@ -0,0 +1,1295 @@
+.boardSettingsTool {
+  padding: 0.9rem;
+  border: 1px solid var(--mantine-color-default-border);
+  border-radius: var(--mantine-radius-md);
+  background: var(--mantine-color-body);
+}
+
+.boardSettingsHeader {
+  min-width: 0;
+}
+
+.boardSettingsTabs :global(.mantine-Tabs-list) {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)